ee/consoleuser/consoleuser_linux.go

```diff
@@ -36,8 +36,9 @@ func CurrentUids(ctx context.Context) ([]string, error) {
 
 	var uids []string
 	for _, s := range sessions {
-		// generally human users start at 1000 on linux
-		if s.UID < 1000 {
+		// generally human users start at 1000 on linux. 65534 is reserved for https://wiki.ubuntu.com/nobody,
+		// which we don't want to count as a current user.
+		if s.UID < 1000 || s.UID == 65534 {
```

Comment:
If we have username here, I'd add a case for that. But only if we already have it. OTOH this is probably good enough for any modern system.

Reply:
I didn't add a check for username because it's not in the struct -- but it is available in the loginctl output, so I can add it to the struct and add the check here.